src/lib_ccx/ccx_encoders_helpers.c

```diff
@@ -31,16 +44,68 @@ static const char *spell_builtin[] =
 	NULL
 };
 
+const char *profane_builtin[] =
+{
+	"arse",
+	"ass",
+	"asshole",
+	"bastard",
+	"bitch",
+	"bollocks",
+	"child-fucker",
+	"Christ on a bike",
+	"Christ on a cracker",
+	"crap",
+	"cunt",
+	"damn",
+	"frigger",
+	"fuck",
+	"goddamn",
+	"godsdamn",
+	"hell",
+	"holy shit",
+	"horseshit",
+	"Jesus fuck",
+	"Jesus Harold Christ",
+	"Jesus wept",
+	"Judas Priest",
```

Comment:
I'm not sure this should be considered as a swear word :p

If there's ever a documentary on https://en.wikipedia.org/wiki/Judas_Priest, this will be annoying ;)